packages/KInlineEdit/KInlineEdit.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    id="editable-wrapper"
+    class="k-inline-edit">
+    <input
+      v-if="isEditing"
+      ref="input"
+      :style="styles"
+      v-model="inputText"
+      class="k-input"
+      @blur="handleSave"
+      @keyup.enter="handleSave">
+    <div
+      id="element-content-wrapper"
+      @click="handleClick">
+      <slot v-if="!isEditing" />
+    </div>
+  </div>
+</template>
+
+<script>
+// Styles we want copied from the element
+const STYLES = {
+  fontSize: 'font-size',
+  fontWeight: 'font-weight',
+  fontFamily: 'font-family',
+  color: 'color',
+  margin: 'margin',
+  padding: 'padding'
+}
+
+export default {
+  name: 'KInlineEdit',
+  data () {
+    return {
+      isEditing: false,
+      inputText: '',
+      styles: {}
+    }
+  },
+  computed: {
+    listeners () {
+      return {
+        ...this.$listeners,
+        blur: e => {
+          this.handleSave(e)
+        }
+      }
+    }
+  },
+
+  mounted () {
+    if (this.$slots.default[0].tag === undefined) {
+      console.warn(`KInlineEdit expects slotted HTML tag.
+
+Example usage:
```

Comment:
this formatting looks off?

Reply:
The formatting has to be like this to look correct in the browser console. This is the same as KToggle https://github.com/Kong/kongponents/blob/master/packages/KToggle/KToggle.js#L34